src/screens/SelectModeScreen.js

```diff
@@ -1,80 +1,69 @@
 import React, { Component } from 'react';
 import {
-  Text,
-  Button,
   View,
   FlatList,
   StyleSheet,
   StatusBar,
   Alert,
   Platform
 } from 'react-native';
-// import Orientation from "react-native-orientation"
-import PeerList from '../components/PeerList'
+import { Container, Content, Button, Text, Grid, Row, Col } from 'native-base';
+import PeerList from '../components/PeerList';
+import styles from "./styles/selectModeScreenStyle";
 
 export default class SelectModeScreen extends Component {
-  // componentWillMount(){
-  //   const initialOrientation = Orientation.getInitialOrientation();
-  //   if (initialOrientation == 'PORTRAIT'){
-  //     Orientation.lockToPortrait();
-  //   } else {
-  //     Orientation.lockToPortrait();
-  //   }
-  // }
   componentDidMount(){
-    // if (Platform.OS != 'Jest') {
-    //   Orientation.lockToPortrait();
-
-    // }
     StatusBar.setHidden(false);
   }
+
   static navigationOptions = {
     title: 'Select Mode'
   }
 
   render() {
     const {navigate} = this.props.navigation;
     return (
-      <View style={styles.container}>
-        <View style={styles.modes}>
-          <Button
-            title='Controller'
-            onPress={() => navigate('ControllerScreen')}
-          />
-          <Button
-            title='Vitals Monitor'
-            onPress={() => navigate('MonitorScreen')}
-          />
-          <Button
-            title='Patient Face'
-            onPress={() => navigate('PatientScreen')}
-          />
-        </View>
-        <View style={styles.peers}>
-          <PeerList data={[{id: '1'}, {id: '2'}, {id: '3'}, {id: '4'}, {id: '5'}, {id: '6'}, {id: '7'}, {id: '8'}, {id: '9'}, {id: '10'}, {id: '11'}, {id: '12'} ]}/>
+      <Container>
```

Comment:
I'm still confused about the usage of row and column. I've always heard it's bad to use grids in CSS/HTML, but if this is the way it's supposed to be and it scales well to tablet and phone and between os then it's probably fine.

Reply:
The grid in native base is an extension of flexbox, not exactly CSS Grid (https://github.com/GeekyAnts/react-native-easy-grid).